### PR TITLE
Update pins (#2167)

### PR DESCRIPTION
**Pull request by rene-dekker**
Pick 5b14c5ef78a25ec2527bd7d7163ffefaf84fb39e

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**
